Addressed comments brought up in peer review

b-sherman opened the pull request:
I decided to remove the multiple linear regression section because it seems beyond the scope of this module. Those images, MLR.png and MLR_POBF.png, can safely be removed from our image folder. I left a mention to it for completeness. I also added captions for all figures and equations to explain what they are.

b-sherman committed:
Addressed comments brought up in peer review

I decided to remove the multiple linear regression section because it seems beyond the scope of this module. Those images, MLR.png and MLR_POBF.png, can safely be removed from our image folder. I left a mention to it for completeness. I also added captions for all figures and equations to explain what they are.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/machinelearningmindset/python-machine-learning?pullRequest=11) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/machinelearningmindset/python-machine-learning?pullRequest=11) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/machinelearningmindset/python-machine-learning?pullRequest=11) it.</sub>

mergu commented on docs/source/content/overview/linear-regression.rst:
```diff
@@ -6,81 +6,83 @@ Linear Regression
-**Linear regression** is a technique used to analyze a **linear relationship** between **input/independent/x** variables and a single **output/dependent/y** variable. A **linear relationship** means that the data points tend to follow a straight line. **Simple linear regression** involves only a single input variable.
-
-.. image:: _img/LR.png
+**Linear regression** is a technique used to analyze a **linear relationship** between **input** variables and a single **output** variable. A **linear relationship** means that the data points tend to follow a straight line. **Simple linear regression** involves only a single input variable.
```
Since we already have this sentence in the above paragraph, we should use another opener. Something like 

```
For any data set, we can find a linear relationship between all the points. A linear relationship means ... 
There are several different types of linear regression - one such type is simple linear regression. Simple linear ... 
Let's take a look at an example linear relationship with the following data points:
[Plot]
```

Generally this is what I mean by transitions - it's better to move into a fact/image before stating/displaying it outright

mergu commented on docs/source/content/overview/linear-regression.rst:
```diff
@@ -6,81 +6,83 @@ Linear Regression
-Our goal is to find the line that best models the path of the data points called a line of best fit [*Equation 1*].
+.. figure:: _img/LR.png
+   
+   *Figure 1*. A sample data set with a linear relationship
```
I'm not sure how these figure captions will turn out on the site, but in github they're a bit hard to distinguish from other text. Can we make them a smaller font or bold or some other way to distinguish them?

mergu commented on docs/source/content/overview/linear-regression.rst:
```diff
@@ -6,81 +6,83 @@ Linear Regression
-.. image:: _img/MLR_POBF.png
-
-Adding extra input variables just means that we’ll need to find more weights. For this exercise, we will only consider a simple linear regression.
+Note that we could have more than one input variable. In this case, we call it **multiple linear regression**. Adding extra input variables just means that we’ll need to find more weights. For this exercise, we will only consider a simple linear regression.
```
Could link out to a tutorial about multiple linear regression, this is up to you though. Something like:
```
If you're interested in learning about multiple linear regression, you can take a look at this guide:
https://towardsdatascience.com/super-simple-machine-learning-by-me-multiple-linear-regression-part-1-447800e8b624
```

mergu reviewed (commented):
Just had a few recommendations, another general one we should follow is line length limits. That hasn't been formally discussed but I generally split each sentence into a newline. If there's no empty whitespace between sentences then they will show up in the same paragraph so you should be able to do this without changing the appearance of the doc at all

mergu reviewed (changes requested):
Could you also remove any images that aren't being used in this PR

b-sherman committed:
Updated linear regression text

+Removed summary because it didn't feel necessary for the short document.
+Capped lines at 79 characters per Sphinx and PEP-8 style with the exception of the URL also per PEP-8.
+Added an introductory sentence for each figure and equation in some way. This makes some of the captions seem repetitive but captions are certainly needed for any sort of image used.
+No way that I could find to change caption size without editing the CSS for the document. There is also no out-of-the-box format like *** to bold and italicize so the captions are simply bold now.
+Chose not to include an external link for multiple regression because that is well above the stats expected from a user.

b-sherman committed:
Deleted MLR.png

Image is no longer needed.

b-sherman committed:
Deleted MLR_POBF.png

Image is no longer needed.

b-sherman commented:
Changed the rst file based on the recommendations and removed unused images.

astorfi commented:
Thank you ... This section looks good.
